Bencodex/Types/Binary.cs

```diff
@@ -197,6 +198,16 @@ public IEnumerable<byte[]> EncodeIntoChunks()
             yield return ((IKey)this).EncodeAsByteArray();
         }
 
+        public void EncodeToStream(Stream stream)
+        {
+            var value = Value;
+            string len = value.Length.ToString(CultureInfo.InvariantCulture);
+            byte[] lenBytes = Encoding.ASCII.GetBytes(len);
+            stream.Write(lenBytes, 0, lenBytes.Length);
+            stream.WriteByte(CommonVariables.Separator[0]);
+            stream.Write(value, 0, value.Length);
```

Comment:
If we write bytes into the stream directly, IMO There is no problem to keep the original value from pollution. So how about using `_value` directly to avoid useless `Array.Copy`?

Reply:
You're right. I fixed it. https://github.com/planetarium/bencodex.net/pull/32/files#diff-4d39369c607a55ca85b9383d14c5e1e974d357e7dd69d82925947c0cf389bbe7R203